python/cudf/cudf/dataframe/index.py

```diff
@@ -604,6 +610,9 @@ def __init__(self, values, **kwargs):
 
         assert isinstance(values, columnops.TypedColumnBase), type(values)
 
+    def __contains__(self, item):
+        return item in self._values
```

Comment:
Either leave this and make the base Index class raise NotImplementedError, or inherit this from the base Index class